otx/algorithms/detection/adapters/mmdet/config_utils.py

```diff
@@ -0,0 +1,394 @@
+"""Collection of utils for task implementation in Detection Task."""
```

Comment:
Why don't we put this file into adapters/mmdet/utils folder? I think it's better to put all files in subfolders of mmdet. So make it remain no files in mmdet except for` __init__.py` like anomaly.

Reply:
I moved it inside `mmdet/utils`. Thanks for the nice comment.

Reply:
It looks great!

Reply:
Please resolve comments that have been resolved. :)